### ISSUE TITLE
ARC1_TO_SOLENOID1_OUT_POS is not connect with U5100

### ISSUE THREAD

**Issue by EnmanuelLuis02** [Phoneboard/phoneboard#92]
ARC1_TO_SOLENOID1_OUT_POS is not connect with U5100.

att: Esantos

**Comment by iTesla13** [comment]
Hello, which model you refer to?
I have look in last software all seems connected from iP8 to iPx, please clarify what model exactly and what software version you use.

**Comment by EnmanuelLuis02** [comment]
iphone X
________________________________

**Comment by iTesla13** [comment]
Yes it is connecting from U5100 > D2 pin to dock connector and 3 caps as per schematic, check the last version of Phoneboard, maybe you use old version with that mistake.
![Screenshot (1722)](https://user-images.githubusercontent.com/74606332/99434147-1f3cd880-2917-11eb-81f4-0f3a680a84d1.png)

**Comment by EnmanuelLuis02** [comment]
very  good thanks

________________________________

**Comment by iTesla13** [comment]
If is solved you can close this case.
